logging/colors/colorize_unix.go

```diff
@@ -5,11 +5,21 @@ package colors
 
 import "fmt"
 
-// EnableColor is a no-op function for non-windows systems because we know that they support ANSI escape codes
-func EnableColor() {}
+var enabled = true
+
+// EnableColor is a simple function for non-windows systems because we know that they support ANSI escape codes
+func EnableColor() { enabled = true }
+
+// DisableColor is a simple function for non-windows systems because we know that they support ANSI escape codes
+func DisableColor() { enabled = false }
```

Comment:
Let's update the inline comments for `EnableColor` and `DisableColor` so that they describe the functionality a bit more. I know it's a bit pedantic but want to make sure that we follow best practices on this.